json_schema/kitchen-sink.yaml

```diff
@@ -1,160 +1,427 @@
-# This demonstrates a file configuration that falls out naturally from the SDK configuration options defined in the specification.
-# The configuration shown is a "kitchen sink" example, with all the available options shown for demonstration purposes.
-
-# User defined reusable elements
-shared:
-  # Define anchor for OTLP export args for reuse across signals
-  otlp_args: &otlpArgs
-    protocol: grpc
-    endpoint: http://localost:4317
-    headers:
-      # TODO: Replace with environment variable when parsing to avoid storing secret in plain text
-      api-key: 1234
-    compression: gzip
-    timeout_millis: 30_000
-
-# SDK configuration.
+# include version specification in configuration files to help with parsing and schema evolution.
+scheme_version: 0.1
```

Comment:
(minor) should this be `schema` rather than `scheme`?

Reply:
There was a discussion around this https://github.com/MrAlias/otel-schema/issues/15, and the decision to call this scheme instead of schema was to avoid confusion with the schema url